xbmc/addons/Scraper.cpp

```diff
@@ -447,7 +447,7 @@ CScraperUrl CScraper::NfoUrl(const std::string &sNfoContent)
 
   // parse returned XML: either <error> element on error, blank on failure,
   // or <url>...</url> or <url>...</url><id>...</id> on success
-  for (unsigned int i=0; i < vcsOut.size(); ++i)
+  for (size_t i=0; i < vcsOut.size(); ++i)
```

Comment:
Dont see entire block but foreach?

Reply:
I checked but i is used for formatting in all the loops I changed